frontend/src/components/NavBar.module.scss

```
@@ -76,3 +76,13 @@
   @extend .transform-transition;
   color: $COLOR_BLACK40;
 }
+
+.userInfoContainer {
+  @extend .typography-body-small;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-left: -2px;
+  margin-right: 10px;
+  gap: 2px;
+}
```

```suggestion
  gap: 2px;
  .memberIcon svg {
    width: 18px !important;
  }
}
```

Ikonit tuntuu inasen isoilta